spec/integration/manifest.spec.yml

```
@@ -410,6 +410,33 @@ spec:
                 - access
           deployment_modes:
             $ref: "#/definitions/deployment_modes"
+          links:
+            description: List of links related to policy template.
+            type: array
+            minItems: 1
+            items:
+              type: object
+              additionalProperties: false
+              properties:
+                title:
+                  description: Link title
+                  type: string
+                url:
+                  description: Link url
+                  type: string
+                type:
+                  description: Type of link. `next_steps` for a regular link, `action` for a link button.
```

Maybe we should be more explicit about this, while keeping this agnostic of how this is implemented in the UI.
```suggestion
                  description: Type of link. `next_steps` for links to locations that can be relevant right after configuring the policy. `action` for actions that can be performed while the policy is in use.
```